.cirrus.yml

```diff
@@ -205,15 +241,15 @@ task:
     memory: 256Mi
 
   depends_on:
-    - freebsd-13
-    - netbsd-postgres
-    - openbsd-postgres
-    - bullseye
-    - sid
-    - sid-newkernel
-    - sid-newkernel-uring
-    - windows-ci-vs-2019
-    - windows-ci-mingw64
+    - 'Building VM Image: ${FREEBSD_13}'
+    - 'Building VM Image: ${NETBSD_POSTGRES}'
+    - 'Building VM Image: ${OPENBSD_POSTGRES}'
+    - 'Building VM Image: ${BULLSEYE}'
+    - 'Building VM Image: ${SID}'
+    - 'Building VM Image: ${SID_NEWKERNEL}'
+    - 'Building VM Image: ${SID_NEWKERNEL_URING}'
+    - 'Building VM Image: ${WINDOWS_CI_VS_2019_VM}'
+    - 'Building VM Image: ${WINDOWS_CI_VS_MINGW64_VM}'
```

Comment:
We could define an alias for the tasks to make this a bit less annoying. See:
https://cirrus-ci.org/guide/writing-tasks/#task-execution-dependencies